domain/schema/model/sql/0017-machine.sql

```diff
@@ -73,3 +73,8 @@ CREATE TABLE machine_filesystem (
     REFERENCES storage_filesystem (uuid),
     PRIMARY KEY (machine_uuid, filesystem_uuid)
 );
+
+CREATE TABLE machine_requires_reboot
+(
+    machine_uuid TEXT NOT NULL PRIMARY KEY
+);
```

Comment:
This needs to be sqlfulff'ed - See PR as an example https://github.com/juju/juju/pull/17449

Nit: It might be worth adding a created at, just for debugging purposes.